README.md

```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
 
 Ailly's best feature is rapid prompt engineering iteration. By keeping your prompts in snippets on the file system, you can make very fine-grained changes to your prompt and immediately see the difference in the output. You can also use all of your normal source control tooling to track changes over time: both your changes, and those from the LLM.
 
+## VSCode extension
+
+The VSCode extension provides several commands.
+
+`Run` runs Ailly on a file. You can activate it with either the right-click context menu or the command palette.
+
+- When you run Ailly from the context menu, it runs on the file with its parent folder as the root
+- When you run Ailly from the command palette, it runs on the current active file (or does nothing if there's no active editor).
+
+`Run All` runs Ailly on a folder and all files below it in the file system.
+
+`Continue` sends the response as an `Assistant` message, to have the LLM continue generating.
+This is especially useful when a request ends before the LLM is 'finished', for instance, asking it to repeat and edit a long document.
+
+`Clean` and `Clean All` on a file or folder clean Ailly details from the file. This includes normalizing the head and removing the response contents.
+
+`Edit` runs from the command palette on the current active file (or does nothing if there's no active editor).
```

Comment:
Again, "active editor" > "active file" or maybe "file selected" or "file open in the editor"?